tox.ini

```diff
@@ -231,13 +252,12 @@ deps =
     component_tastypie-tastypielatest: django-tastypie
     component_tastypie-tastypielatest: django<4.1
     component_tastypie-tastypielatest: asgiref<3.7.1  # asgiref==3.7.1 only suppport Python 3.10+
-    coroutines_asyncio-{py37,py38,py39,py310,py311,py312}: uvloop
+    coroutines_asyncio-{py37,py38,py39,py310,py311,py312,py313}: uvloop
```

Comment:
Was `py313` meant to be added here? This isn't being tested on 3.13 above due to the `uvloop` dependency.

Reply:
It was added for when we enable the actual tox environment in the future.